AsaApi/main.cpp

```diff
@@ -63,6 +63,43 @@ void OpenConsole()
 	SetConsoleOutputCP(CP_UTF8);
 }
 
+void DeleteOldLogFiles()
+{
+	const std::string config_path = AsaApi::Tools::GetCurrentDir() + "/config.json";
```

Comment:
Using the config example I mentioned in config.json. This format allows for the config file to be missing and not break existing installs of the API while maintaining the current functionality of not pruning logs. Accessing the config has you have it currently will crash if they do not update the config.

`if (config.value("DeleteOldLogs", nlohmann::json::object()).value("Enabled", false) == false)
   return;
   
   const int maxAgeInSeconds = config.value("DeleteOldLogs", nlohmann::json::object()).value("MaxAgeInHours", 24) * 3600
`